### ISSUE TITLE
RP fails on Python3 running 00_getting_started.py

### ISSUE THREAD

**Issue by karahbit** [radical-cybertools/radical.pilot#2039]
Local Machine: Jetstream Atmosphere VM that @iparask created with Docker and gsissh.

radical-stack: 
```
 python               : 3.7.6
  pythonpath           : 
  virtualenv           : /home/karahbit/ve-entk3

  radical.entk         : 1.0.0
  radical.pilot        : 1.0.0
  radical.saga         : 1.0.0
  radical.utils        : 1.0.0
```

[radical_sandbox_session.zip](https://github.com/radical-cybertools/radical.pilot/files/4119145/radical_sandbox_session.zip)
[local_session.zip](https://github.com/radical-cybertools/radical.pilot/files/4119146/local_session.zip)

**Comment by karahbit** [comment]
Update: Following the log files, it has something to do with openMPI on Jetstream VM.

@krb09 sent me the following command to reinstall openMPI and it worked after that:
```
sudo apt-get install --reinstall openmpi-bin libopenmpi-dev
```

I will leave this open for review and for somebody else to close the issue.

**Comment by mturilli** [comment]
This is odd, example_00 should not require openmpi

**Comment by andre-merzky** [comment]
This should be fixed in #2041.

**Comment by andre-merzky** [comment]
#2041 was merged and fixes this.
